src/coreclr/src/debug/CMakeLists.txt

```diff
@@ -3,3 +3,6 @@ add_subdirectory(ildbsymlib)
 add_subdirectory(ee)
 add_subdirectory(di)
 add_subdirectory(shim)
+if(CLR_CMAKE_HOST_LINUX OR CLR_CMAKE_HOST_WIN32)
```

Comment:
Why is this only Linux and Windows?

Reply:
I didn't think we will ever support single-file on MacOS. Now as far as building the runtimeinfo library, I could make some changes to GetModuleIndex and the build system to support it if it important.

Reply:
Publishing an app as a single-file is currently supported on MacOS, but it is not a very interesting case, and there are no known users.

In the short term, we'll continue to support single-file apps on MacOS, but there are no plans to build the super-host (that is, coreclr will be a separate file on disk beside the app).  

So, the change is fine as-is. I was just curious if MacOS was excluded because we don't need the library, or if there were any other challenges. Thanks.